libpod/runtime.go

```diff
@@ -164,14 +164,22 @@ var (
 		CgroupManager: CgroupfsCgroupsManager,
 		HooksDir:      hooks.DefaultDir,
 		StaticDir:     filepath.Join(storage.DefaultStoreOptions.GraphRoot, "libpod"),
-		TmpDir:        "/var/run/libpod",
+		TmpDir:        getDefaultTmpDir(),
 		MaxLogSize:    -1,
 		NoPivotRoot:   false,
 		CNIConfigDir:  "/etc/cni/net.d/",
 		CNIPluginDir:  []string{"/usr/libexec/cni", "/usr/lib/cni", "/opt/cni/bin"},
 	}
 )
 
+func getDefaultTmpDir() string {
+	if os.Getuid() == 0 {
+		return "/var/run/libpod"
+	}
+
+	return filepath.Join(os.Getenv("XDG_RUNTIME_DIR"), "podman", "tmpdir")
```

Comment:
(`"libpod"` or `"podman"`, then? Or are both paths pre-existing by now?)

Reply:
I've seen both used but "libpod" seems to be more correct (as it is already used for TmpDir), going to change it.